instance/migrations/0076_openedxappserver__is_active.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is ok.. I worked around it (see [comments](https://github.com/open-craft/opencraft/pull/181/files/d27df1bfc8b646eaf47644d7b8af9bc6b467257f#r101799806)).